sympy/solvers/tests/test_solveset.py

```diff
@@ -32,6 +33,27 @@
 n = Symbol('n', real=True)
 
 
+def test_dummy():
```

Comment:
Am 10.01.2015 um 14:47 schrieb Sergey B Kirpichev:
>> @@ -32,6 +33,27 @@
>>   n = Symbol('n', real=True)
>>
>>
>> +def test_dummy():
>
> No.  At least this should to separate commit.

Fixed.

(Sending directly because Github does not show the line anymore because 
it was squashed out during rebase.)